lib/builderator/tasks/packer.rb

```
@@ -203,6 +203,13 @@ def images
         end
       end
 
+      def copy_image(region, params)
+        Retryable.retryable(:sleep => lambda { |n| 4**n }, :tries => 4, :on => [Aws::EC2::Errors::ServiceError]) do |retries, _|
+          say_status :error, 'Error copying image', :red if retries.positive?
+          Util.ec2(region).copy_image(parameters)
```

Do we want any error handling if the number of `retries == tries`?

---

What do you have in mind?  If we have used up our tries this should raise the exception which indicates an error.  If we haven't succeeded before we run out of tries we likely never will.

---

> What do you have in mind? If we have used up our tries this should raise the exception which indicates an error. If we haven't succeeded before we run out of tries we likely never will.

Unhiding this.